evansmusomi/burger-builder
# Persists Auth State via Local Storage

*What this does*
- Stores the token, userId and expiry time in local storage to auto sign in users

*How to verify*
- A user should be able to sign in and stay signed in despite refreshing the browser
- A user should be required to sign in after they log out
